helm/broker-app/templates/deploymentconfig.yaml

```diff
@@ -27,6 +27,18 @@ spec:
       labels:
         app: {{ .Values.global.name }}
         deploymentconfig: {{ .Values.global.name }}
+    annotations:
+        vault.hashicorp.com/agent-inject: 'true'
+        vault.hashicorp.com/agent-inject-secret-broker-token: a03c8f-{{ .Values.global.envlable }}/{{ .Values.global.envtag }}-token
+        vault.hashicorp.com/agent-inject-template-broker-token: |
+          {{`{{- with secret `}}"a03c8f-{{ .Values.global.envlable }}/{{ .Values.global.envtag }}-token" {{` }}
+          export BROKER_TOKEN="{{ .Data.data.BROKER_TOKEN }}"
+          export VAULT_TOKEN="{{ .Data.data.VAULT_TOKEN }}"
+          {{ end `}} }}       
+        vault.hashicorp.com/agent-inject-token: 'true'
+        vault.hashicorp.com/auth-path: auth/k8s-CLUSTER
+        vault.hashicorp.com/namespace: {{ .Release.Namespace }}
+        vault.hashicorp.com/role: a03c8f-{{ .Values.global.envlable }}
```

Comment:
Should we be extracting all license plate references from .Release.Namespace?

Reply:
Yes, all license plate has been extracted from .Release.Namespace with the updated PR.